python/pyspark/util.pyi

```diff
@@ -0,0 +1,33 @@
+#
```

Comment:
This has been already ported, though I am not sure if we're going to consider it a part of the API.